### PR TITLE
add more headings to the github pr template

### PR DESCRIPTION
**Pull request by thedavidmeister**
proposal for an extended/simpler pr template

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+## PR summary
+
+## testing/benchmarking notes
+
+( if any manual testing or benchmarking was/should be done, add notes and/or screenshots here )
+
+## followups
+
+( any new tickets/concerns that were discovered or created during this work but aren't in scope for review here )
+
```
Comment (zippy) [inline .github/pull_request_template.md]: Here's a suggestion for the docs section.  @Connoropolous any other suggestions?
```suggestion
## documentation

- [ ] this change has no user-facing documentation that needs updating
- [ ] rust-doc comments have been added
- [ ] md-book documentation updates have been added
```

Comment (Connoropolous) [reply .github/pull_request_template.md]: This works!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+## changelog
+
+Please check one of the following, relating to the [CHANGELOG](https://github.com/holochain/holochain-rust/blob/develop/CHANGELOG.md)
```
Comment (Connoropolous) [inline .github/pull_request_template.md]:
```suggestion
Please place an x into one of the following (to display it as a checkmark), relating to the [CHANGELOG](https://github.com/holochain/holochain-rust/blob/develop/CHANGELOG.md)
```

File: .github/pull_request_template.md (hunk 1 above)
Comment (Connoropolous) [reply .github/pull_request_template.md]: Note to self and others, we had a discussion with dhtnetwork discussing how devs could also have the option of delivering documentation for new code in the form of video, if writing is a challenge/pain/undesirable

**Pull request merged**